src/UKHO.ERPFacade.Common/Models/TableEntities/EESEventTable.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Azure;
+using Azure.Data.Tables;
+
+namespace UKHO.ERPFacade.Common.Models.TableEntities
+{
+    public class EESEventTable : ITableEntity
```

Comment:
Please use [ExcludeFromCodeCoverage] for this class

Reply:
Done